src/toRedactor.tsx

```diff
@@ -362,7 +403,18 @@ export const toRedactor = (jsonValue: any,options?:IJsonToHtmlOptions) : string
           delete attrsJson['content-type-uid']
           attrsJson['sys-style-type'] = allattrs['display-type']
           delete attrsJson['display-type']
-        } else if (attrsJson['type'] === "asset") {
+        } 
+        else if (attrsJson["display-type"]) {
+          const styleObj = jsonValue["attrs"]["style"];
```

Comment:
can we have some fallback values and optional chaining here

Reply:
This is done